app/views/common/BreadcrumbComponent.vue

```diff
@@ -0,0 +1,36 @@
+<style scoped>
+    a {
+         color: #065e73;
+     }
+</style>
+
+<template>
+    <div class="breadcrumbs rtl-allowed">
+        <a v-if="links.length > 0" :href="links[0].href">{{ links[0].i18n ? $t(links[0].i18n) : links[0].text }}</a>
+        <span v-if="links.length > 1">
+            <span v-if="links[1].href" class="spl spr">></span>
+            <span v-else class="spl spr"> > {{ links[1].i18n ? $t(links[1].i18n) : links[1].text }}</span>
+            <a v-if="links[1].href" :href="links[1].href">{{ links[1].i18n ? $t(links[1].i18n) : links[1].text }}</a>
+        </span>
+        <span v-if="links.length > 2">
+            <span v-if="links[2].href" class="spl spr">></span>
+            <span v-else class="spl spr"> > {{ links[2].i18n ? $t(links[2].i18n) : links[2].text }}</span>
+            <a v-if="links[2].href" :href="links[2].href">{{ links[2].i18n ? $t(links[2].i18n) : links[2].text }}</a>
+        </span>
+    </div>
+</template>
+
+<script>
+  export default {
+    props: {
+      /**
+      * Each item represents a link with optional href, i18n translated description or just plain text
+      * {Array} links
```

Comment:
Trying out some JSDoc (http://usejsdoc.org/about-getting-started.html) here! This might be wrong - a bit unsure how to properly describe elements in each object in an array.